<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Option</class>
 <widget class="QDialog" name="Option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Option</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/Option.png</normaloff>:/Icons/Option.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QGroupBox" name="OptionGroupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string notr="true">GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="LanguageLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Language :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="LanguageComboBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Automatic (System Language)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>English</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Persian</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spanish</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>49</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="OptionButtonBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidgetOption">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>148</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icons/Option.png</normaloff>:/Icons/Option.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Language</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icons/language.png</normaloff>:/Icons/language.png</iconset>
      </property>
     </item>
    </widget>
   </item>
  </layout>
  <zorder>OptionButtonBox</zorder>
  <zorder>listWidgetOption</zorder>
  <zorder>OptionGroupBox</zorder>
 </widget>
 <tabstops>
  <tabstop>listWidgetOption</tabstop>
  <tabstop>LanguageComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
